addons/account_edi_facturx/data/facturx_templates.xml

```diff
@@ -112,7 +112,7 @@
 
                 <!-- Document Headers. -->
                 <rsm:ExchangedDocument>
-                    <ram:ID t-esc="record.ref"/>
+                    <ram:ID t-esc="record.name"/>
```

Comment:
separate PR.  (same for the print)

Reply:
Oopsie (just tested some other PR on my branch).